### PR TITLE
Certificate design 96

### PR DESCRIPTION
**Pull request by sanket-jadhav07**

### CODE REVIEW

**Review by laxmikant-gaykhee — CHANGES_REQUESTED** [review]
- style.css file add in css folder 
-  images add in img folder
-  add preview.index file
-  add font folder

**Comment by sanketsj07** [comment]
changed all request

File: SubmittedAssignments/Sanket_Jadhav/preview.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+		<!--Content of Certificate-->
+		<div class="certificate-content">
+			<div class="outerborder">
+				<div class="innerborder">
+					<div class="title">C E R T I F I C A T E</div>
+					<div class="recttop"></div>
+					<div class="rectleft"></div>
+					<div class="recttop1"></div>
+					<div class="recttop2"></div>
+					<div class="recttop3"></div>
```
Comment (laxmikant-gaykhee) [inline SubmittedAssignments/Sanket_Jadhav/preview.html:32]: the design class should be out side in content class (add in div class="certificate-background")

File: SubmittedAssignments/Sanket_Jadhav/preview.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+</head>
+
+<body>
+	<!--Certificate Wrapper-->
+	<div class="certificate-wrapper">
+				
+
+		<!--Certificate background-->
+		<div class="certificate-background">
+
```
Comment (laxmikant-gaykhee) [inline SubmittedAssignments/Sanket_Jadhav/preview.html:19]: -Add Background Div here

File: SubmittedAssignments/Sanket_Jadhav/preview.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+					<div class="title">C E R T I F I C A T E</div>
+					<div class="recttop"></div>
+					<div class="rectleft"></div>
+					<div class="recttop1"></div>
+					<div class="recttop2"></div>
+					<div class="recttop3"></div>
+					<div class="subtitle"><b>O F &nbsp A P P R E C I A T I O N</b></div>
+					<div class="line"></div>
+					<div class="line1"></div>
+					<div class="radius"></div>
```
Comment (laxmikant-gaykhee) [inline SubmittedAssignments/Sanket_Jadhav/preview.html:36]: -there div also add in background class

**Review by KalpeshApnasite — CHANGES_REQUESTED** [review]
remove certificate-content from index.html

**Review by KalpeshApnasite — CHANGES_REQUESTED** [review]
rename the folder name

File: SubmittedAssignments/Sanket_Jadhav/css/style.css
Comment (KalpeshApnasite) [inline SubmittedAssignments/Sanket_Jadhav/css/style.css:1]: follow the folder name format

**Review by KalpeshApnasite — CHANGES_REQUESTED** [review]

**Pull request merged**